src/script/campaign/mod/vco-listeners.lua

```diff
@@ -139,6 +193,18 @@ local function check_vco_kho_exiles_of_khorne_skarbrand_rank_40(character)
 	end
 end
 
+local function check_vco_nor_wulfrik_chaos_allegiance_levels(pooled_resource)
+  local FINISHED_ONE_GOD_ALREADY = cm:get_saved_value("vco_nor_wul_aligned_to_god") or false;
```

Comment:
Is any of this needed? Wouldn't this work if I just switched the `true` to `false` in the listener so it fires once and once only on the first occurrence of a chaos god reaching 100 pooled resource?